src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/CyclomaticComplexityCheck.java

```diff
@@ -99,50 +99,166 @@
  * &lt;module name="CyclomaticComplexity"/&gt;
  * </pre>
  * <p>
- * To configure the check with a threshold of 15:
+ * Example:
+ * </p>
+ * <pre>
+ * class CyclomaticComplexity {
+ *   // Cyclomatic Complexity = 12
+ *   public void foo() { // 1, function declaration
+ *     if (a == b) { // 2, if
+ *       fun1();
+ *     } else if (a == c // 3, if
+ *       &amp;&amp; a == d) { // 4, &amp;&amp;
+ *       if (c == e) { // 5, if
+ *         fun2();
+ *       }
+ *     } else if (a == c // 6, if
+ *       || a == g) { // 7, ||
```

Comment:
please add "operator" to just operator signs, I mean to make it like "|| operator"